packages/eth-rpc-cache/src/rpc.ts

```diff
@@ -64,7 +58,10 @@ export const createEthRpcCache = function (
     try {
       const strategyName = strategyResolver[method]?.(method, params)
       if (strategyName) {
-        return cachesByStrategy[strategyName](method, params).then(clone)
+        return cachesByStrategy[strategyName](method, params).then(c =>
+          // can't inline on .then(structuredClone), TS fails to infer
+          structuredClone(c)
```

Comment:
`structuredClone` was added in Node 17. The `engines.node` shall be updated. Why didn't `eslint-plugin-node` detected the problem?